tests/test_bundle.py

```diff
@@ -90,6 +91,68 @@ def _test_bundle_get(self, replica: Replica):
             self.assertEqual(resp_obj.json['bundle']['files'][0]['uuid'], "ce55fd51-7833-469b-be0b-5da88ebebfcd")
             self.assertEqual(resp_obj.json['bundle']['files'][0]['version'], "2017-06-16T193604.240704Z")
 
+    @testmode.standalone
+    def test_bundle_get_paging(self):
+        bundle_uuid = "7f8c686d-a439-4376-b367-ac93fc28df43"
+        version = "2019-02-21T184000.899031Z"
+        with override_bucket_config(BucketConfig.TEST_FIXTURE):
+            handle = Config.get_blobstore_handle(Replica.aws)
+            manifest = json.loads(handle.get(
+                Replica.aws.bucket,
+                f"bundles/{bundle_uuid}.{version}"
+            ))
+            expected_files = manifest['files']
+
+        with self.subTest("Should be able to use a variety of valid paging with the same result"):
+            self._test_bundle_get_paging(Replica.aws, expected_files, 11)
```

Comment:
breaking each of these into their own subtest based on the parameters used. It will give you better test results by narrowing down which specific cases that failed. 
**Bonus:** Since you can't nest subtests, You could make each subtest a test case, each parameter set a subtest, and use the currents subtest string as a docstring for the test case.

Reply:
fixed